plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/internal/EditorsMessages.java

```diff
@@ -24,13 +24,16 @@ public class EditorsMessages extends NLS {
 	public static String dialog_struct_attribute_edit_page_header_edit_attribute;
 	public static String dialog_struct_label_text_name;
 	public static String dialog_struct_label_text_properties;
+	public static String dialog_struct_unique;
```

Comment:
Please, fix tabs and incorrect indentation level (should be 4) in your strings in this file.